server/slack-bot.js

```diff
@@ -41,14 +41,26 @@ This bot demonstrates many of the core features of Botkit:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 require('isomorphic-fetch');
 const queryString = require('query-string');
+const port = process.env.PORT || process.env.VCAP_APP_PORT || 3000;
 const slackBotToken = process.env.SLACK_BOT_TOKEN || 
                       process.env.VCAP_APP_SLACK_BOT_TOKEN;
-
-if (!slackBotToken) {
+const slackBotHostRoute = process.env.VCAP_APP_SLACK_BOT_HOST_ROUTE;
```

Comment:
just an observation.. why do the others check both VCAP_APP_FOO and FOO? should this one also?